caf-logging-logback-tomcat/src/main/java/com/github/cafapi/logging/logback/tomcat/CAFTomcatAccessLogPatternLayout.java

```diff
@@ -0,0 +1,29 @@
+/*
+ * Copyright 2019-2020 Micro Focus or one of its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.github.cafapi.logging.logback.tomcat;
+
+import ch.qos.logback.access.PatternLayout;
+import java.util.Map;
+
+public final class CAFTomcatAccessLogPatternLayout<E> extends PatternLayout
+{
+    @Override
+    public Map<String, String> getDefaultConverterMap()
+    {
+        defaultConverterMap.put("loglevel", CAFTomcatAccessLogLevelConverter.class.getName());
```

Comment:
Is it possible to use the `conversionRule` rather than creating custom encoder and layout (i.e. the same way that the `SanitizeConverter` and `MaybeJsonMessageConverter` classes are brought in)?

Reply:
The conversionRule for loglevel is working fine. I have tested with tomcat image. Submitting this Pr for re review.